airflow/jobs/scheduler_job.py

```diff
@@ -664,6 +665,10 @@ def _execute(self) -> None:
 
         try:
             self.executor.job_id = self.id
+            self.executor.callbacks_sink = PipeCallbacksSink(
```

Comment:
```suggestion
            self.executor.callback_sink = PipeCallbacksSink(
```

`callbacks_sink` looks slightly odd to me as an English speaker.

Reply:
Renamed to callback_sink + in other places, including class names.